python/rascaline/utils.py

```diff
@@ -20,3 +22,225 @@ def _call_with_growing_buffer(callback, initial=1024):
             else:
                 raise
     return buffer.value.decode("utf8")
+
+
+def convert_old_hyperparameter_names(hyperparameters, mode="librascal"):
```

Comment:
```suggestion
def translate_soap_hyperparameters(hyperparameters, mode="librascal"):
```

since this is soap-specific